components/normalizer/tests/tests.rs

```diff
@@ -1618,14 +1618,46 @@ fn test_is_normalized_up_to() {
         assert!(nfkc.is_normalized(&normalized));
     };
 
-    // todo: UTF16 tests?
+    // Check a string of UTF-16 code units is normalized up to where is_normalized_up_to reports
+    let check_utf16 = |input: &[u16]| {
+        // Check nfd
+        let up_to = nfd.is_normalized_utf16_up_to(input);
+        let (head, tail) = input.split_at(up_to);
```

Comment:
thought: maybe `is_normalized_up_to` should return this tuple, because `split_at` will be an additional bounds check and a potential panic. The length can be recovered by doing `.0.len()` if that's needed.

Reply:
Filed as https://github.com/unicode-org/icu4x/issues/5217 . Thanks.